content/doc/developer/cli/index.adoc

```diff
@@ -9,5 +9,7 @@ references:
   title: CLI user documentation in the User Handbook
 - url: https://wiki.jenkins.io/display/JENKINS/Jenkins+CLI
   title: CLI documentation on the Jenkins wiki
+- url: https://github.com/jenkins-zh/jenkins-cli
```

Comment:
It would make sense to consider hosting on jenkinsci beforehand

Reply:
Can I just keep it stay for now? I hope this could encourage more Chinese contributors. Maybe I'm wrong. But I just want to give it a shot.